lib/takeaway.rb

```diff
@@ -0,0 +1,35 @@
+class Takeaway
+  def initialize
+    @message = []
+  end
+  
+  def notification(deliver)
+    @message << "Thank you! Your order was placed on #{Time.now.strftime("%d/%m/%Y %H:%M")} and will be delivered before #{(Time.now + 1 * 60 * 60).strftime("%H:%M")}"
```

Comment:
In order to cut down on the amount of code within this method you could potentially have a separate method which is responsible for returning the expected delivery time.